### PR TITLE
Minimum number of swaps required for bracket balancing added

### PR DESCRIPTION
**Pull request by Avantika782**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes
### Description
Have added the program for Minimum number of swaps for bracket balancing in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue number: #4826
Pull request: #6025

### CODE REVIEW

**Comment by Avantika782** [comment]
@ankitaggarwal23 Please check my PR and guide me if it needs any correction.

**Comment by gat786** [comment]
Change the title of PR. it currently just says your username outloud.

File: C-Plus-Plus/other/Minimum_number_of_swaps_for_bracket_balancing.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/*
+
+We are given a string of 2N characters
```
Comment (gat786) [inline C-Plus-Plus/other/Minimum_number_of_swaps_for_bracket_balancing.cpp]: take all this at the bottom of code file. While there is no particular reason to do so. But as a personal liking I try to keep all the most important things in a code file at the top or the most reachable point. all the information or not so important pieces hidden in the bottom.

**Review by gat786 — CHANGES_REQUESTED** [review]

**Comment by Avantika782** [comment]
@gat786 Please review the changes and let me know if it requires any correction.

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -574,6 +574,7 @@ _add list here_
 - [Concatenate given strings](other/n_string_concatenate.cpp)
 - [Decimal to Gray code Conversion](other/Decimal_to_gray.cpp)
 - [Trapping Rainwater](other/trappingRainwater.cpp)
+- [Minimum number of swaps for bracket balancing](other/Minimum_number_of_swaps_for_bracket_balancing)
```
Comment (gat786) [inline C-Plus-Plus/README.md]: This link won't work. It doesn't have the extension applied.

**Comment by gat786** [comment]
Rest LGTM :)

File: C-Plus-Plus/README.md (hunk 2 above)
Comment (Avantika782) [reply C-Plus-Plus/README.md]: Please review it. @gat786

Comment (Avantika782) [reply C-Plus-Plus/README.md]: OKhay , if the rest is okhay;-))

**Review by ankitaggarwal23 — COMMENTED** [review]
Code plagiarized from [here](https://www.geeksforgeeks.org/minimum-swaps-bracket-balancing/)

**Comment by Avantika782** [comment]
@ankitaggarwal23 I want to correct my mistakes. Sir, can please reassign it to me.

**Comment by Avantika782** [comment]
@ankitaggarwal23 Sir I want to correct my mistakes. Can you please reassign it to me.

**Pull request closed without merging**